### PR TITLE
OCPQE-18684: Add optional presubmits which run QE-written e2e tests against the CCO repo

### PR DESCRIPTION
**Pull request by fxierh**
The presubmits added:
- Are optional, i.e. does not block PRs (against the CCO repo) from being merged
- Run against QE's account and budget
- Run QE-written e2e test cases that cover some CCO-specific functionalities
- Have a `test` chain which finishes in 10 minutes

Once this PR is merged we'll start debugging and further stabilizing our test cases so they all pass reliably. 

Depends on: 
- [CCO#645](https://github.com/openshift/cloud-credential-operator/pull/645) and backports

### CODE REVIEW

**Comment by fxierh** [comment]
/pj-rehearse pull-ci-openshift-cloud-credential-operator-master-e2e-aws-qe

**Comment by fxierh** [comment]
/pj-rehearse pull-ci-openshift-cloud-credential-operator-master-e2e-aws-qe

**Comment by fxierh** [comment]
/pj-rehearse pull-ci-openshift-cloud-credential-operator-master-e2e-aws-qe pull-ci-openshift-cloud-credential-operator-release-4.11-e2e-aws-qe pull-ci-openshift-cloud-credential-operator-release-4.12-e2e-aws-qe pull-ci-openshift-cloud-credential-operator-release-4.13-e2e-aws-qe pull-ci-openshift-cloud-credential-operator-release-4.14-e2e-aws-qe pull-ci-openshift-cloud-credential-operator-release-4.15-e2e-aws-qe pull-ci-openshift-cloud-credential-operator-release-4.16-e2e-aws-qe

**Comment by fxierh** [comment]
@YuLi517 @jianping-shu 

/assign @jstuever 
Would you please take a look ? Thanks.

**Comment by fxierh** [comment]
/pj-rehearse pull-ci-openshift-cloud-credential-operator-release-4.13-e2e-aws-qe

**Comment by dlom** [comment]
/pj-rehearse pull-ci-openshift-cloud-credential-operator-release-4.12-e2e-aws-qe
/pj-rehearse pull-ci-openshift-cloud-credential-operator-release-4.16-e2e-aws-qe
/pj-rehearse pull-ci-openshift-cloud-credential-operator-release-4.14-e2e-aws-qe
/pj-rehearse pull-ci-openshift-cloud-credential-operator-release-4.15-e2e-aws-qe
/pj-rehearse pull-ci-openshift-cloud-credential-operator-master-e2e-aws-qe

**Comment by dlom** [comment]
/approve
/hold
Holding pending a lgtm from @jstuever (and all rehearsals passing)

**Comment by fxierh** [comment]
/pj-rehearse pull-ci-openshift-cloud-credential-operator-release-4.11-e2e-aws-qe

**Comment by dlom** [comment]
/pj-rehearse pull-ci-openshift-cloud-credential-operator-release-4.14-e2e-aws-qe

**Comment by dlom** [comment]
/pj-rehearse pull-ci-openshift-cloud-credential-operator-release-4.15-e2e-aws-qe

**Comment by dlom** [comment]
/pj-rehearse pull-ci-openshift-cloud-credential-operator-release-4.16-e2e-aws-qe

**Comment by dlom** [comment]
/pj-rehearse pull-ci-openshift-cloud-credential-operator-master-e2e-aws-qe

**Review by jstuever — APPROVED** [review]
/lgtm

**Comment by fxierh** [comment]
/pj-rehearse pull-ci-openshift-cloud-credential-operator-release-4.16-e2e-aws-qe pull-ci-openshift-cloud-credential-operator-release-4.14-e2e-aws-qe pull-ci-openshift-cloud-credential-operator-release-4.15-e2e-aws-qe pull-ci-openshift-cloud-credential-operator-master-e2e-aws-qe

**Comment by fxierh** [comment]
/unhold

All rehearsals passed.

**Pull request merged**
